Problem: missing support `query_denom_by_name` of NFT

add NFT api for query_denom_by_name(NFT)

```

296
cluster = <pystarport.cluster.ClusterCLI object at 0x7f6694b48c70>
297

298
    def test_query_denom_by_name(cluster):
299
        denomname = "testdenomname"
300
        singer1_addr = cluster.address("signer1")
301
>       response = cluster.query_denom_by_name(denomname)
302
E       AttributeError: 'ClusterCLI' object has no attribute 'query_denom_by_name'
```

--- a/pystarport/cluster.py
+++ b/pystarport/cluster.py
@@ -575,6 +575,9 @@ class ClusterCLI:
     def query_nft(self, denomid="mydenomid", i=0):
         return self.cosmos_cli(i).query_nft(denomid)
 
+    def query_denom_by_name(self, denomname="mydenomname", i=0):
+        return self.cosmos_cli(i).query_denom_by_name(denomname)
+
     def create_nft_token(
         self,
         from_addr,
@@ -684,8 +687,7 @@ def init_devnet(
         if use_ledger:
             acct = cli.create_account_ledger(account["name"])
         else:
-            mnemonic = account.get("mnemonic")
-            acct = cli.create_account(account["name"], mnemonic=mnemonic)
+            acct = cli.create_account(account["name"])
         vesting = account.get("vesting")
         if not vesting:
             cli.add_genesis_account(acct["address"], account["coins"])

--- a/pystarport/cosmoscli.py
+++ b/pystarport/cosmoscli.py
@@ -861,6 +861,20 @@ class CosmosCLI:
             )
         )
 
+    def query_denom_by_name(self, denomname):
+        return json.loads(
+            self.raw(
+                "query",
+                "nft",
+                "denom-by-name",
+                denomname,
+                output="json",
+                home=self.data_dir,
+                chain_id=self.chain_id,
+                node=self.node_rpc,
+            )
+        )
+
     def create_nft_token(self, from_addr, to_addr, denomid, tokenid, uri, fees):
         return json.loads(
             self.raw(

